### PR TITLE
feat: add a few function to utils for count

### LINKED ISSUE #23: Make Counter Example with `core` and `utils` package
- [ ] Create example as below
- [x] add `wordCount` method to `@djsp/utils`

```
import { getWordCount } from '@djsp/utils'
import { EditorContainer, Plugin, Editor } from '@djsp/core'

<EditorContainer>
  <Plugin>{({ editorState }) => getWordCount(editorState)}</Plugin>
  <Editor />
</EditorContainer>
```

### PR DESCRIPTION
**Pull request by freedomlang**
Resolve #23

### CODE REVIEW

File: examples/counter/src/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  render() {
+    return (
+      <div>
+        <EditorContainer
+          editorState={this.state.editorState}
+          onChange={this.onChange}>
+          <div>Char count: {getCharCount(this.state.editorState)}</div>
+          <div>Word count: {getWordCount(this.state.editorState)}</div>
+          <div>Line count: {getLineCount(this.state.editorState)}</div>
```
Comment (juliankrispel) [inline examples/counter/src/index.js]: Make this a separate component `WordCountPlugin.js`

```
export default () => <Plugin>{({ editorState }) => <Fragment>
          <div>Char count: {getCharCount(this.state.editorState)}</div>
          <div>Word count: {getWordCount(this.state.editorState)}</div>
          <div>Line count: {getLineCount(this.state.editorState)}</div>
</Fragment>
```

And import it to `index.js`

File: packages/utils/dist/index.es.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { AtomicBlockUtils, RichUtils, EditorState, Modifier } from 'draft-js';
+import punycode from 'punycode';
```
Comment (juliankrispel) [inline packages/utils/dist/index.es.js]: How big is this dependency? Also, it needs to be added to `external` in the rollup config

Comment (juliankrispel) [reply packages/utils/dist/index.es.js]: why do we need this exactly?

File: packages/utils/dist/index.js [hunk 3]
```diff
@@ -2,7 +2,10 @@
 
 Object.defineProperty(exports, '__esModule', { value: true });
```
Comment (juliankrispel) [inline packages/utils/dist/index.js]: `dist` folder should be ignored (let's check the .gitignore file

File: packages/utils/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@djsp/utils",
-  "version": "0.0.3",
+  "version": "0.0.4",
```
Comment (juliankrispel) [inline packages/utils/package.json]: this will update after a rebase

File: packages/utils/src/index.js [hunk 5]
```diff
@@ -9,6 +9,8 @@ import {
   Modifier,
 } from 'draft-js'
 import type DraftEntityInstance from 'draft-js/lib/DraftEntityInstance'
+// eslint-disable-next-line
```
Comment (juliankrispel) [inline packages/utils/src/index.js]: shouldn't need to use the eslint comment here

File: packages/utils/src/index.js [hunk 6]
```diff
@@ -127,3 +129,24 @@ export function hasEntity(
   const entity = this.getCurrentEntity(editorState)
   return entity && entity.getType() === entityType
 }
+
+export function getCharCount(editorState: EditorState): number {
```
Comment (juliankrispel) [inline packages/utils/src/index.js]: does this pass flow?

File: packages/utils/dist/index.es.js (hunk 2 above)
Comment (freedomlang) [reply packages/utils/dist/index.es.js]: This dependency used for handle unicode characters. 

It's size is 142B(MINIFIED). Still need to added it into `external`?

File: packages/utils/src/index.js (hunk 6 above)
Comment (freedomlang) [reply packages/utils/src/index.js]: Currently, `@djsp/utils` can not pass flow. Looks like missing some files in flow-lib

File: packages/utils/src/index.js (hunk 5 above)
Comment (freedomlang) [reply packages/utils/src/index.js]: Eslint will throw error about `deprecated-api`. So i add `eslint-disable-next-line node/no-deprecated-api` in the next commit

File: packages/utils/package.json (hunk 4 above)
Comment (freedomlang) [reply packages/utils/package.json]: Ah, after `git rebase master`, the version updated to `0.1.5`. Not sure if it is right

**Pull request merged**